src/electionguard/voting/encrypter.c

```diff
@@ -12,9 +12,9 @@
 #include "serialize/voting.h"
 #include "sha2-openbsd.h"
 #include "voting/message_reps.h"
-#include "voting/num_ballots.h"
 
-uint64_t Voting_num_ballots = 0;
+// count of ballots encrypted with this encrypter
+uint64_t encrypted_ballot_count = 0;
```

Comment:
Naming convention. This global variable can easily be confused by a local variable when reading code